packages/ember/src/components/mktg/feature-list.gts

```diff
@@ -0,0 +1,52 @@
+import Component from '@glimmer/component';
+import type { ComponentLike } from '@glint/template';
+import type { TOC } from '@ember/component/template-only';
+
+const columnMap = {
+  '1': 'g-col-12',
+  '2': 'g-col-6',
+  '3': 'g-col-4',
+  '4': 'g-col-3',
+  '6': 'g-col-2',
+  '12': 'g-col-1',
+};
+
+interface FeatureSignature {
+  Args: {
+    icon?: string;
+    text?: string;
+    class: string;
+  };
+  Element: HTMLParagraphElement;
+}
+
+export interface MktgFeatureListSignature {
+  Element: HTMLDivElement;
+  Blocks: {
+    features: [ComponentLike<typeof Feature>];
+    label: [];
+    default?: [];
+  };
+  Args: {
+    columns: keyof typeof columnMap;
+  };
+}
+
+const Feature: TOC<FeatureSignature> = <template>
+  <p class={{@class}} ...attributes>
+    <span class="me-2 fw-bold bi {{@icon}}" />{{@text}}
```

Comment:
I noticed in the figma designs, some of the feature lists are bolded numbers, like for channel guides 
![image](https://github.com/user-attachments/assets/e8d7577a-c33c-43ee-8877-f0e6a75a0612)
Do you think it would be worth adding an optional yield to the span to render the channel numbers and bold them? It may not be necessary, just trying to think of all the options!

Reply:
It could also be a param instead of a yield I'd think, whatever works best